tasks/Debian-tasks.yml

```diff
@@ -58,6 +58,13 @@
     dest: "{{ checkmk_agent_authkey_dest }}"
   tags:
     - checkmk-agent
+- name: Delete directory - local | FILE
```

Comment:
What are the deletions for?

Reply:
Lets say,

you had the plugin xyz in the main plugin folder and now you want to move that to the plugin/async folder.
With the new function you would list it under the dict "checkmk_agent_plugin_checks_async".
Now you would have in the main folder still  the plugin xyz and also in the plugin/async folder.

I hope you understand me.

Because of that i add the optional "purge" for local & plugin folder.